### PR TITLE
prep for 4.2.0 release and helm chart update

### PR DESCRIPTION
**Pull request by lucasreed**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -15,12 +15,12 @@
 package main
 
 import (
-	"github.com/fairwindsops/polaris/cmd/polaris"
+	cmd "github.com/fairwindsops/polaris/cmd/polaris"
```
Comment (lucasreed) [inline main.go:18]: My editor kept wanting to change this so even though the module at that directory is named `cmd` this makes the below code more clear I think.

Comment (rbren) [reply main.go:18]: 👍

**Review by rbren — APPROVED** [review]

**Review by rbren — COMMENTED** [review]
Oh - README.md needs the new version too

**Comment by rbren** [comment]
Also if you woudn't mind updating docs/changelog.md

**Review by rbren — APPROVED** [review]

**Pull request merged**